flow_action_components/GetOpportunityPartners/force-app/main/default/classes/GetOpportunityPartnersAction.cls

```diff
@@ -0,0 +1,132 @@
+/**
+* @author       : Tamar Erlich
+* @date         : October 07, 2020
+* @description  : Invocable method that given opportunityId and accountId, returns a list of opportunity partners to display on opportunity plan record page
+* Note          : This class is called by the Flow.
+* *************************************************
+*       <runTest>GetOpportunityPartnersActionTest<runTest>
+* *************************************************
+* @History
+* -------
+* VERSION | AUTHOR                | DATE               | DESCRIPTION
+* 1.0     | Tamar Erlich          | October 07, 2020   | Created
+**/
+global with sharing class GetOpportunityPartnersAction {
+
+    // Attributes passed in from the Flow
+    global class Requests {
+
+        @InvocableVariable(label='OpportunityId' required=true)
+        global String opportunityId;
+        
+        @InvocableVariable(label='AccountId' required=true)
+        global String accountId;
+    }
+
+    // Attributes passed back to the Flow
+    global class Results {
+
+        public Results() {
+            
+        }
+        @InvocableVariable
+        global String errors;
+
+        @InvocableVariable
+        global String successful;
+
+        @InvocableVariable
+        public List<OpportunityPartnersWrapper> outputCollection;
+
+        @InvocableVariable
+        public String outputString;
+        
+    }
+
+    // custom exception class
+    global class InvocableActionException extends Exception {}
+
+    // Expose this Action to the Flow
+    @InvocableMethod
+        global static List <Results> get(List<Requests> requestList) {
+            // initialize variables
+            Results response = new Results();
+            List<Results> responseWrapper = new List<Results>();
+            String errors;
+            String success = 'true';
+            String stringOutput;
+            String accountId = requestList[0].accountId;
+            String opportunityId = requestList[0].opportunityId;
+            List<OpportunityPartner> opportunityPartners = new List<OpportunityPartner>();
+            List<OpportunityPartnersWrapper> partners = new List<OpportunityPartnersWrapper>();
```

Comment:
not a good variable name. too general. For a data type like this, I'd use something like oppPartnerWrapperSet or oppPartnerWrappers.